assets/floormaps/hall/HallFloorX.js

```diff
@@ -8,17 +8,15 @@ export function HallFloorX(props) {
   const [floorNumber, setFloorNumber] = React.useState("");
   const to = props.to;
   const from = props.from;
+  const mobility = props.mobility;
 
   const floorSelected = async () => {
     let name = await AsyncStorage.getItem("floorSelected");
-    setFloorNumber(name);
+    name == 1 ? setFloorNumber(1) : setFloorNumber(name);
```

Comment:
is name a number or a value?

Reply:
ill change the name of the variable to `floor`